scss/tech.scss

```diff
@@ -52,3 +51,28 @@
   grid-column: colum 2;
   grid-row: row 2;
 }
+
+@media only screen and (max-width: 1024px) {
+  .grid-container {
+    grid-template-columns: 100%;
+
+    .tech-container {
+      flex-direction: column;
+      grid-column-start: 1;
+      grid-row-end: 3;
+      justify-self: center;
+    }
+
+    .tech {
+      border-top: 2px $color-green solid;
+    }
+  }
+
+  .tech-main:nth-child(2) {
```

Comment:
I dont know what is this for, tech-main has 2 childs? And you only apply style to the second?